runtime/fail_byt.c

```diff
@@ -158,43 +164,57 @@ CAMLexport void caml_array_bound_error(void)
 CAMLexport void caml_raise_out_of_memory(void)
 {
   check_global_data("Out_of_memory");
-  caml_raise_constant(Field(caml_global_data, OUT_OF_MEMORY_EXN));
+  caml_raise_constant(Field(caml_global_data,
```

Comment:
Fix formatting; match trunk. Also, other definitions that are in this file.

Reply:
Fixed in https://github.com/ocaml-multicore/ocaml/commit/7d2d9e0a2cf1969b1ca0e621bca4cc2534b88d91